### ISSUE TITLE
geom_forecast() grayscale colours become red

### ISSUE THREAD

**Issue by robjhyndman** [robjhyndman/forecast#445]
Colours sometimes incorrect:

```R
library(fpp)
library(ggplot2)
fit <- holt(air, h=5) 
fit2 <- holt(air, damped=TRUE, h=5) 

autoplot(air, series="Data") +
  geom_forecast(fit, PI=FALSE, series="Holt's method") +
  geom_forecast(fit2, PI=FALSE, series="Damped Holt's method") +
  ggtitle("Forecasts from Holt's method") +
  xlab("Year") + ylab("Air passengers in Australia (thousands)") +
  scale_color_manual(
    values=c("Data"="black","Holt's method"="blue","Damped Holt's method"="red"),
    breaks=c("Data","Holt's method", "Damped Holt's method"))
```

**Comment by mitchelloharawild** [comment]
Grayscale colours by default have a hue of 0, which is the hue for "red".
Saturation/value values are ignored, to produce appealing/controlled colours.

I'm working on a new way to create the continuous/discrete colour combination for the plot, which should be better supported internally by ggplot, and should also allow us to create a legend for the forecast interval's level.

If this is urgent, I can do a quickfix via an exception.

**Comment by robjhyndman** [comment]
OK. It is not urgent.

**Comment by mitchelloharawild** [comment]
Fixed in #451 
```R
library(ggplot2)
fc <- forecast(mdeaths)
autoplot(mdeaths) + geom_forecast(fc, series="mdeaths") + scale_colour_manual(values = c("mdeaths" = "grey"))
```
![rplot](https://cloud.githubusercontent.com/assets/16127127/21633153/c106366a-d2a0-11e6-83cf-eeaba94a6ec0.png)
